dist/resources/virtualAccounts.js

```diff
@@ -82,12 +82,10 @@ module.exports = function (api) {
         return Promise.reject(ID_REQUIRED_MSG);
       }
 
-      var data = {
-        status: 'closed'
-      };
+      var data = {};
```

Comment:
I don't think we need to make changes in the dist folder. It should be updated when the build is run

Reply:
@dhwanilvyas i have an doubt , does it build automatically when user install this package from npmjs ?

Reply:
+1

@ankitdas13 Dist should be updating upon building the package

Reply:
@sidag95 i have removed those changes